package.json

```diff
@@ -1,39 +1,40 @@
 {
-    "name": "expo-drizzle-studio-plugin",
-    "version": "0.1.1",
-    "description": "A new DevTools plugin created by create-dev-plugin",
-    "main": "build/index.js",
-    "types": "build/index.d.ts",
-    "sideEffects": false,
-    "scripts": {
-        "build": "expo-module build",
-        "pack": "npm pack",
-        "build:all": "expo-module prepare && cd webui && npx expo export -p web --output-dir ../dist",
-        "clean": "expo-module clean",
-        "prepare": "expo-module prepare",
-        "prepublishOnly": "expo-module prepare && expo-module prepublishOnly && cd webui && npx expo export -p web --output-dir ../dist",
-        "web:dev": "cd webui && npx expo start -w",
-        "web:export": "cd webui && npx expo export -p web --output-dir ../dist"
-    },
-    "keywords": [
-        "expo",
-        "devtools"
-    ],
-    "files": [
-        "build",
-        "dist",
-        "expo-module.config.json"
-    ],
-    "license": "MIT",
-    "dependencies": {
-        "expo-sqlite": "^15.0.3"
-    },
-    "devDependencies": {
-        "expo": "~52.0.0",
-        "expo-module-scripts": "^3.1.0",
-        "typescript": "^5.1.3"
-    },
-    "peerDependencies": {
-        "expo": "*"
-    }
+  "name": "expo-drizzle-studio-plugin",
+  "version": "0.1.1",
+  "description": "A new DevTools plugin created by create-dev-plugin",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "sideEffects": false,
+  "scripts": {
+    "build": "expo-module build",
+    "pack": "npm pack",
+    "build:all": "expo-module prepare && cd webui && npx expo export -p web --output-dir ../dist",
+    "clean": "expo-module clean",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepare && expo-module prepublishOnly && cd webui && npx expo export -p web --output-dir ../dist",
+    "web:dev": "cd webui && npx expo start -w",
+    "web:export": "cd webui && npx expo export -p web --output-dir ../dist"
+  },
+  "keywords": [
+    "expo",
+    "devtools"
+  ],
+  "files": [
+    "build",
+    "dist",
+    "expo-module.config.json"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "@op-engineering/op-sqlite": "^10.1.0",
+    "expo-sqlite": "^15.0.3"
```

Comment:
Would it be better to make `@op-engineering/op-sqlite` and `expo-sqlite` peer dependencies?

Having both in `dependencies` would cause both to be installed. This increases the bundle size unnecessarily and may cause native build failures (see [OP SQLite's documentation](https://ospfranco.notion.site/Gotchas-bedf4f3e9dc1444480fc687d8917751a#01a0d7e42e804a6b967fba916e4f96e5)).

Reply:
Sure, that is right. Thanks!